src/services/shardQuerySplitting.ts

```diff
@@ -167,7 +168,9 @@ function splitQueriesByStreamShard(
           done();
           return;
         }
-        nextRequest();
+        if (!combineResponsesWorker) {
```

Comment:
lets make sure we test this after disabling web workers, in an iframe, and in a mobile webview

Reply:
Yeah, definitely. It needs to run if workers are not available.